.gitignore

```diff
@@ -8,6 +8,7 @@ ve
 dist
 build
 .idea
+env
```

Comment:
do not track virtualenv folder

Reply:
I'm not sure why your virtualenv folder is in the source code folder, and I don't think you can always assume someone is going to call this folder `env`.

Reply:
It is very convenient when `env` folder is in project. 
By the way, the folders `env`,` venv`, `ENV` are always added to gitignore, if you use auto-generation of gitignore for `python` projects. So I think it's make sense because a lot of developers creates `env` folder inside project.

Reply:
See #540